torch/ao/quantization/_pt2e/utils.py

```diff
@@ -127,3 +128,15 @@ def _rearrange_weight_observer_for_decomposed_linear(
     model.graph.eliminate_dead_code()
     model.graph.lint()
     model.recompile()
+
+def _build_node_name_to_scope(model: GraphModule) -> Dict[str, Tuple[str, type]]:
```

Comment:
nit: build --> get might be clearer I feel

Reply:
@jerryzh168 is this actively used? If this is set to empty dict, would the workflow still work?

Reply:
Let's do that separately. Here I'm just refactoring to reduce duplicate code